packages/workflow/src/Interfaces.ts

```diff
@@ -1631,3 +1633,7 @@ export type PublicInstalledNode = {
 	latestVersion: string;
 	package: PublicInstalledPackage;
 };
+
+export interface INodeExecutionMetaData extends INodeExecutionData {
```

Comment:
Hey @brianinoa we're moving away from the `I` prefix for interfaces. Can I suggest a new name? `NodeExecutionWithMetadata` sounds better IMO. Can you please apply this small refactor? Thanks!